libs/bixarena/api-description/src/components/schemas/Model.yaml

```diff
@@ -9,19 +9,45 @@ properties:
     type: string
     description: URL-friendly unique slug for the model.
     example: 'my-awesome-model'
+  alias:
+    type: string
+    nullable: true
+    description: Alternative name or alias for the model.
+    example: 'awesome-model-v2'
   name:
     type: string
     description: Human-readable name of the model.
     example: 'My Awesome Model'
-  license:
+  organization:
     type: string
     nullable: true
-    description: License under which the model is released.
-    example: 'Apache-2.0'
+    description: Organization that developed or maintains the model.
+    example: 'OpenAI'
+  license:
+    $ref: '../schemas/LicenseType.yaml'
```

Comment:
Use the same name for the property and schema for consistency. either `licence` / `License.yaml` or `licenseType` / `LicenseType`.